openlibrary/plugins/upstream/addtag.py

```diff
@@ -11,20 +11,22 @@
 from infogami.infobase.client import ClientException
 from infogami.utils import delegate
 
-from openlibrary.plugins.openlibrary.processors import urlsafe
-from openlibrary.i18n import gettext as _
+from openlibrary.plugins.openlibrary.processors import urlsafe  # XXX : unused
+from openlibrary.i18n import gettext as _   # XXX : unused
 import logging
 
 from openlibrary.plugins.upstream import spamcheck, utils
 from openlibrary.plugins.upstream.models import Tag
 from openlibrary.plugins.upstream.addbook import get_recaptcha, safe_seeother, trim_doc
 from openlibrary.plugins.upstream.utils import render_template
 
+# XXX : Nothing being logged here
```

Comment:
Plus, `name` that is passed to `getLogger` doesn't appear to follow existing conventions.

Reply:
Our conventions appear to exist entirely in my imagination...  Not sure what we want to do here (remove logger altogether, update `name`, nothing at all), but I'm committing to using `__name__` as the name of any new loggers that I add.

Reply:
Remove